src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/22.diff.sql

```diff
@@ -0,0 +1,2501 @@
+﻿/*************************************************************
+    Stored procedure to Acquire export jobs
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     Acquires export jobs.
```

Comment:
Silly question: I noticed that there are stored procs in this .diff file that seem to be from previous schema versions. Why do we need to have them here as well?

Reply:
22.diff.sql file contains 26 stored proc which popped up as differences between snapshot vs diff database. There are no structural changes for any of the SP. This file only includes formatting and styling of SP's to avoid differences

Reply:
Ahh makes sense. Thank you!